### DISCUSSION TITLE
Holdbrew Mod

### DISCUSSION THREAD

**Discussion by MrGenie151** [ppy/osu#22436] [Ideas] [answered]
I think it would be cool if there was a mania mod where you don't have to time when you release hold notes, with a 0.90x or 0.95x multiplier decrease.
Half the time I have the "Hold Off" mod enabled because I don't like timing when I let go.

**Comment by mk56-spn** [comment, marked as answer]
A multiplier that lenient would potentially see this dominating extremely LN heavy maps . Which would be the last thing people would want presumably.  Otherwise this is a decent idea I guess

**Reply by MrGenie151** [reply]
then multiplier 0.80x or 0.85x

**Reply by mk56-spn** [reply]
sounds fine. mind explaining the name?

**Reply by MrGenie151** [reply]
i heard it from my cousin about a rhythm game that i can remember it's name.
